modules/client-configuration/pages/clients-debian.adoc

```diff
@@ -3,174 +3,119 @@
 
 This section contains information about registering Salt clients running {debian} operating systems.
 
-{productname} supports for {debian} 9 Stretch and 10 Buster clients is experimental.
+[IMPORTANT]
+====
+{suse} does not provide support for {debian} operating systems.
+{productname} allows you to manage {debian} clients, but support is not provided.
+Using {productname} to manage {debian} clients is experimental.
+These instructions have been tested on {debian}{nbsp}9 Stretch and {debian}{nbsp}10 Buster.
+Do not rely on them in a production environment.
+====
+
+Bootstrapping can be used with {debian} clients for performing initial state runs, and for profile updates.
+
 
-Bootstrapping is supported for starting {debian} clients and performing initial state runs such as setting repositories and performing profile updates.
 
 == Prepare to Register
 
 Some preparation is required before you can register {debian} clients to the {productname} Server.
 
-[IMPORTANT]
-====
-Debian 9 requires `apt-transport-https` before the registration. Make sure it is installed on the client otherwise registration will fail:
+If you are using Debian{nbsp}9, install the ``apt-transport-https`` package on the client before you attempt to register.
+On the client, at the command prompt, as root, run:
 
 ----
 apt install apt-transport-https
 ----
 
-Ensure DNS is correctly configured and provides an entry for the client, otherwise configure the `/etc/hosts` files on both the uyuni server and the client with the appropriate entries.
-
-The client has to have NTP synchronised correctly with the Uyuni server before the registration
-====
+Ensure DNS is correctly configured and provides an entry for the client.
+Alternatively, you can configure the ``/etc/hosts`` files on both the {productname} Server and the client with the appropriate entries.
 
+The client must have NTP synchronized correctly with the {productname} server before registration.
```

Comment:
I'd just say: "... must have _time_ synchronized correctly."  We usually do not configure the server as an NTP time server.